view/frontend/templates/hyva/bfcache/handler.phtml

```diff
@@ -0,0 +1,173 @@
+<?php declare(strict_types=1);
+
+use MageOS\ThemeOptimization\ViewModel\BfCache as BfCacheViewModel;
+use Magento\Framework\Escaper;
+use Magento\Framework\View\Helper\SecureHtmlRenderer;
+
+/** @var BfCacheViewModel $bfcacheConfig */
+/** @var Escaper $escaper */
+/** @var SecureHtmlRenderer $secureRenderer */
+
+$bfcacheConfig = $block->getData('bfcache_config');
+?>
+
+<?php
```

Comment:
```suggestion

```

Reply:
we already update this in the latest commit